pontoon/base/models.py

```diff
@@ -192,6 +204,69 @@ def latest_activity(self, project=None):
         return translations.latest_activity()
 
 
+@receiver(pre_save, sender=Locale)
+def create_locale_translators_group(sender, instance,  raw, **kwargs):
```

Comment:
[According to the docs](https://docs.djangoproject.com/en/1.8/topics/signals/#receiver-functions), signals should only accept `sender` and `kwargs` in case signal arguments change over time.